### ISSUE TITLE
Narrator gives no information of the pop-up getting opened when any input other than the specifies input is entered.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/WinUI-Gallery#161]
**Describe the bug**

**Pre-requisite :** 
Turn on Narrator (Ctrl+Win+Enter).
Set Narrator in item mode.

**Steps to reproduce the bug**

1. Launch XAML Controls Gallery.
2. Activate Implicit Transition from All controls page.
3. In the Set Opacity textbox, enter any value outside the range of 0.0 to 1.0 or enter any character and observe. 

**Expected behavior**

- When any value outside the range of 0.0 to 1.0 is entered, a pop-up window  "Input must be between 0 and 1"gets opened, so Narrator should announce the pop-up information.
- When any character is entered, a pop-up "The input must be a number" gets opened, So Narrator's focus should go the pop-up and it should announce the pop-up information.

**Actual behavior**

- Narrator remains silent in both the cases.
- Also, in item navigation user cannot go to the pop-up to read its content.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/63796102-40507e00-c8ba-11e9-9a89-53d5b0d08cf7.png)

User Impact:  Visually impaired users will not be able to know that pop-up is getting opened since narrator remains silent.

**Version Info**

Gallery version (found on Settings page): 

App Version: 1.2.9.0

Windows 10 version:
- [x] Insider Build (xxxxx)
- [ ] May 2019 Update (18362)
- [ ] October 2018 Update (17763)
- [ ] April 2018 Update (17134)
- [ ] Fall Creators Update (16299)
- [ ] Creators Update (15063)
- [ ] Anniversary Update (14393)

Device form factor:
- [x] Desktop
- [ ] Mobile
- [ ] Xbox
- [ ] Surface Hub
- [ ] IoT

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_workitems/edit/23304820/)
